plugins/woocommerce/client/admin/client/shared/sidebar-layout.scss

```diff
@@ -0,0 +1,193 @@
+/**
+ * Shared sidebar layout styles
+ * Used by Launch Your Store (LYS) and Customize Your Store (CYS)
+ *
+ * This file contains common styles for sidebar layouts to avoid duplication
+ */
+
+// Mixin for container layout
+@mixin sidebar-layout-container {
+	display: flex;
+	flex-direction: row;
+	background-color: #fcfcfc;
+	min-height: 100vh;
+
+	@media screen and (max-width: 782px) {
+		flex-direction: column;
+	}
+}
+
+// Mixin for mobile header
+@mixin mobile-header {
+	.mobile-header {
+		display: none;
+
+		@media screen and (max-width: 782px) {
+			display: flex;
+			align-items: center;
+			padding: 0.5rem;
+			background: #fff;
+			border-bottom: 1px solid #dcdcde;
+			gap: 0.5rem;
+
+			&__back-button {
+				display: flex;
+				justify-content: center;
+				align-items: center;
+				min-width: 36px;
+				height: 36px;
+				padding: 0;
+
+				svg {
+					fill: $gray-900;
+				}
+			}
+
+			&__title {
+				font-size: 1rem;
+				font-weight: 600;
+				line-height: 24px;
+				color: $gray-900;
+				margin: 0;
+				flex: 1;
+			}
+		}
+	}
+}
+
+// Mixin for sidebar styling
+@mixin sidebar-container {
+	width: 380px;
+	display: flex;
+	flex-direction: column;
+	height: 100%;
+
+	@media screen and (max-width: 782px) {
+		display: none;
+	}
+
+	@media screen and (min-width: 768px) and (max-width: 1024px) {
+		width: 280px;
+	}
+
+	.woocommerce-edit-site-layout__hub {
+		width: 380px;
+		height: 64px;
+	}
+
+	.woocommerce-edit-site-site-hub {
+		align-items: center;
+		display: flex;
+		gap: 8px;
+		justify-content: space-between;
+	}
+
+	.woocommerce-edit-site-site-hub__view-mode-toggle-container {
+		height: 64px;
+		padding: 16px 12px 0 0;
+		color: #ccc;
+		width: auto;
+	}
+
+	.woocommerce-edit-site-sidebar-navigation-screen__title-icon {
+		align-items: center;
+		background-color: transparent;
+		padding-top: 16px;
+		padding-bottom: 0;
+		margin-bottom: 8px;
+		gap: 0;
+		width: 348px;
+		z-index: 2;
+	}
+
+	.woocommerce-edit-site-layout__view-mode-toggle-icon.edit-site-site-icon {
+		align-items: center;
+		display: flex;
+	}
+
+	.woocommerce-edit-site-site-hub__site-title {
+		color: $gray-900;
+		font-size: 0.8125rem;
+		font-style: normal;
+		font-weight: 500;
+		line-height: 20px;
+		margin: 0;
+	}
+
+	.edit-site-site-icon__image {
+		border-radius: 2px;
+		width: 100%;
+		height: auto;
+		object-fit: cover;
+		background: rgba(0, 0, 0, 0);
+	}
+
+	.woocommerce-edit-site-site-hub__view-mode-toggle-container {
+		.woocommerce-edit-site-layout__view-mode-toggle,
+		.woocommerce-edit-site-layout__view-mode-toggle-icon.edit-site-site-icon,
+		.edit-site-site-icon__icon {
+			width: 32px;
+			height: 32px;
+			fill: #ccc;
+		}
+	}
+
+	will-change: auto;
+	padding: 0 16px;
+	overflow-x: hidden;
```

Comment:
Should we move these CSS rules to the top of the selector? Now they are between nested selectors, which feels a bit confusing.

Also, I don't think `will-change: auto` is needed? 🤔